data/strings/strings.txt

```diff
@@ -16326,6 +16326,234 @@
     zh-Hans = 3. 使用应用时选择
     zh-Hant = 3. 使用應用時選擇
 
+  [location_services_disabled_on_device_1]
+    comment = iOS Dialog for the case when the location services disabled on the device in Settings
+    tags = ios
+    en = 1. Open Settings
+    af = 1. Open Instellings
+    ar = 1. افتح الإعدادات
+    az = 1. Parametrləri açın
+    be = 1. Адкрыйце Налады
+    bg = 1. Отваряне на Настройки
+    ca = 1. Obriu la configuració
+    cs = 1. Otevřete Nastavení
+    da = 1. Åbn Indstillinger
+    de = 1. Einstellungen öffnen
+    el = 1. Ανοίξτε τις ρυθμίσεις
+    es = 1. Abre los ajustes
+    et = 1. Ava seaded
+    eu = 1. Ireki ezarpenak
+    fa = 1. بازکردن تنظیمات
+    fi = 1. Avaa asetukset
+    fr = 1. Ouvrir les paramètres
+    he = 1. פתחו את "הגדרות" (Settings)
+    hu = 1. Nyissa meg a Beállításokat
+    id = 1. Buka Pengaturan
+    it = 1. Apri le impostazioni
+    ja = 1. 「設定」を開きます
+    ko = 1. 설정 열기
+    lt = 1. Atidarykite nustatymus
+    mr = १. सेटिंग उघडा
+    nb = 1. Åpne Innstillinger
+    nl = 1. Instellingen openen
+    pl = 1. Uruchom ustawienia
+    pt = 1. Abra as configurações
+    pt-BR = 1. Abra as configurações
+    ro = 1. Deschide setările
+    ru = 1. Откройте Настройки
+    sk = 1. Otvorte Nastavenia
+    sv = 1. Öppna inställningar
+    th = 1. เปิดการตั้งค่า
+    tr = 1. Ayarları Açın
+    uk = 1. Відкрийте «Налаштування»
+    vi = 1. Mở Thiết lập
+    zh-Hans = 1. 打开设置
+    zh-Hant = 1. 開啟設定
+
+  [location_services_disabled_on_device_2]
+    tags = ios
+    en = 2. Select Privacy
+    af = 2. Kies Privaatheid
+    ar = 2. حدد الخصوصية
+    az = 2. Məxfilik seçin
+    be = 2. Абярыце Прыватнасць
+    bg = 2. Изберете Privacy
+    ca = 2. Seleccioneu Privadesa
+    cs = 2. Vyberte možnost Soukromí
+    da = 2. Vælg Privatliv
+    de = 2. Datenschutz auswählen
+    el = 2. Επιλέξτε Απόρρητο
+    es = 2. Selecciona Privacidad
+    es-MX = 2. Seleccione Privacidad
+    et = 2. Valige Privaatsus
+    eu = 2. Hautatu Pribatutasuna
+    fa = 2. Privacy را انتخاب کنید
+    fi = 2. Valitse Tietosuoja
+    fr = 2. Sélectionne Privacy
+    he = 2. בחר פרטיות
+    hi = 2. गोपनीयता का चयन करें
+    hu = 2. Válassza az Adatvédelem
+    id = 2. Pilih Privasi
+    it = 2. Seleziona Privacy
+    ja = 2.プライバシーを選択する
+    ko = 2. 2. 개인정보 보호 선택
+    lt = 2. Pasirinkite Privatumas
+    mr = 2. गोपनीयता निवडा
+    nb = 2. Velg Personvern
+    nl = 2. Selecteer Privacy
+    pl = 2. Wybierz Prywatność
+    pt = 2. Selecciona Privacidade
+    pt-BR = 2. Selecione Privacidade
```

Comment:
pt-BR strings equal to pt strings should be filtered out by translate.py, strange...